qutebrowser/browser/commands.py

```diff
@@ -472,6 +472,9 @@ def navigate(self, where: {'type': ('prev', 'next', 'up', 'increment',
             bg: Open in a background tab.
             window: Open in a new window.
         """
+        # save the pre-jump position in the special ' mark
+        self.set_mark("'")
+
```

Comment:
@rcorre I know it's been a while, but do you remember why you added this here in the `:navigate` command? As far as I can see, `:navigate` will always load a different page, and the `'` mark is local to the current page - so if I'm not mistaken, this will never save anything useful. Am I missing something?

Reply:
Yeah, in retrospect it seems pointless ... can't remember what I was thinking at the time :grin: 

I guess it could matter if `:navigate` clinks on a link that leads to a fragment in the same page? That's the best I've got.